config/crds/azureprovider_v1alpha1_azuremachineproviderstatus.yaml

```diff
@@ -67,7 +64,10 @@ spec:
         metadata:
           type: object
         vmId:
-          description: VMID is the instance ID of the machine created in Azure.
+          description: VMID is the vm ID of the machine created in Azure
```

Comment:
Slight preference for virtual machine instead of VM
```suggestion
          description: VMID is the virtual machine ID of the machine created in Azure.
```